runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/partition/SerializedPartition.java

```diff
@@ -226,4 +241,14 @@ public int getLength() throws IOException {
   public boolean isOffheap() {
     return offheap;
   }
+
+  /**
+   * Releases the off-heap memory that this SerializedPartition holds.
+   */
+  public void release() {
```

Comment:
What do you think about adding a JIRA issue and adding a TODO here as a pointer?

JIRA issue: Consider removing 'transient' uses of SerializedPartition. Use SerializedPartition only for SerializedMemoryStore. FileBlock and ByteOutputContext should write directly to disks/network, and not create intermediary SerializedPartitions and do extra copying. This also simplifies the uses of SerializedPartition#release().

Reply:
Thanks! I just added one.